src/dependencyresolver.js

```diff
@@ -88,9 +93,10 @@ class DependencyResolver {
         try {
           await this.dependencyTracker.resolveTask(m.taskId, m.resolution);
           await m.remove();
+          this.monitor.count('handled-messages-success', 1);
         } catch (err) {
-          debug("[alert-operator] Failed to handle message: %j" +
-                ", with err: %s, as JSON: %j", message, err, err, err.stack);
+          this.monitor.count('handled-messages-error', 1);
+          this.reportError(err, 'warning');
```

Comment:
> Does reporting an error cause things to crash then or is the goal to just catch this error, report it, and not crash?

Nope, but `.emit('error', err)` does... This is just a warning... the loop continues...